docs/source/test.rst

```diff
@@ -3,6 +3,46 @@ Testing with Titanoboa
 
 Titanoboa integrates natively with `pytest <https://docs.pytest.org/>`_ and `hypothesis <https://hypothesis.readthedocs.io/en/latest/quickstart.html>`_. Nothing special is needed to enable these, as the plugins for these packages will be loaded automatically. By default, isolation is enabled for tests - that is, any changes to the EVM state inside the test case will automatically be rolled back after the test case completes.
 
+Getting Started
+-----------------------
+
+To begin writing your smart contract tests with pytest, it is reccomended to create a `tests/` directory to contain your tests. You will also need an `__init__.py` file (empty) within your test directory.
+
+If you are interacting and testing a core set of smart contracts, you will likely want to make a `conftest.py` file to contain testing fixtures. An example is shown below
+
+.. code-block:: python
+
+    from typing import Any, Callable, List
+    import boa
+    import pytest
+
+    @pytest.fixture(scope="session")
+    def accounts() -> List[Any]:
+        return [boa.env.generate_address() for _ in range(10)]
+
+    @pytest.fixture(scope="session")
+    def account(accounts) -> Any:
+        return accounts[0]
+
+    @pytest.fixture(scope="session")
+    def token(account):
+        with boa.env.prank(account):
+            return boa.load('src/mocks/mock_erc20.vy', "Token", "TKO", 18)
+..
+
+The above `conftest.py` creates a test fixture to access both a list of different addresses you can use within your tests, and shows how to instantiate an ERC20 Vyper contract as a test fixture.
+
+Then make a seperate file named `test_erc20.py` within your `tests/` folder, the exact name doesn't matter, but it must start with `test_`. Within this file you can begin to write tests using the fixtures you've defined in `conftest.py`.
+
+.. code-block:: python
+    
+    def test_metadata(token):
+        assert token.name() == "Token"
+        assert token.symbol() == "TKO"
+        assert token.decimals() == 18
+..
+
+Now you have successfully written your first Titanoboa test, to use a fixture, simply add its name as a parameter to a test function and it will be injected into the test. From there, you can call its functions directly, and make assertions about their values.
```

Comment:
```suggestion
Now you have successfully written your first Titanoboa test! To use a fixture, simply add its name as a parameter to a test function and it will be injected into the test. From there, you can call its functions directly, and make assertions about their values.
```